dht/range_streamer.cc

```diff
@@ -276,7 +276,7 @@ future<> range_streamer::stream_async() {
                     if (_nr_rx_added) {
                         sp.request_ranges(source, keyspace, std::move(ranges_to_stream));
                     } else if (_nr_tx_added) {
-                        sp.transfer_ranges(source, keyspace, ranges_to_stream);
+                        sp.transfer_ranges(source, keyspace, std::move(ranges_to_stream));
```

Comment:
As above (and why are the two changes in separate commits - they seem very similar)

Reply:
squashed together in a single commit in v7